pages/about/dataaccess.vue

```diff
@@ -27,7 +27,7 @@
     <v-card class="woudc-note" tile max-width="68%" min-width="0px">
```

Comment:
Do these attributes serve a purpose? `tile`, `max-width` and `min-width`.

I think with the grid system use (`<v-row>` and `<v-col>`) you can eliminate these.

Reply:
`tile` prevents the corners from being rounded, I actually want to stop using it and go back to rounded corners. Not sure if the `min-width="0"` does anything useful either.

As for the `max-width`, I think `<v-card>` has an issue where it automatically fills up the whole screen width. Even the Vuetify documentation page for it had manual `max-width` in every example. I think `max-width` is the most intuitive way of controlling that, especially since we rarely have multiple things on the same row for `v-row`.

Reply:
Expanding on how the use of grid system with breakpoints will help with this: https://vuetifyjs.com/en/components/grids/#row-and-column-breakpoints

One example case that might hinder the user experience is when the user is using a small screen. Setting a fixed `max-width="68%"` on a small screen is pretty small, when it could easily fill up the screen width. 
On the opposite end, an example user with a larger screen, that `max-width="68%"` may look too big for that `v-card`.

Thus, here is an example grid system with breakpoints in use on how it can address the different sizes:
```suggestion
  <v-col sm="12" md="6" lg="4" xl="3">
    <v-card>
      <v-card-title>Card that adjust width on different screen sizes</v-card-title>
    </v-card>
  </v-col>
```